modules/core/04-channel/v2/keeper/packet_test.go

```diff
@@ -255,6 +231,7 @@ func (suite *KeeperTestSuite) TestRecvPacket() {
 	}
 }
 
+/*
 func (suite *KeeperTestSuite) TestWriteAcknowledgement() {
```

Comment:
Commented this function out for now since a separate WriteAcknowledgement function hasn't been written yet for channel/v2

Reply:
it should be here https://github.com/cosmos/ibc-go/blob/2a64425641299167ddbbf2d0ece81dcd8dd2fa4d/modules/core/04-channel/keeper/packet.go#L281

Reply:
whoops wrong link, one sec

Reply:
https://github.com/cosmos/ibc-go/blob/140283e29d8c124d0cdea123afb23eca8a9e3ba0/modules/core/04-channel/v2/keeper/packet.go#L170